tests/smb-version-keyword/README.md

```diff
@@ -0,0 +1,4 @@
+TEST
+====
+
+Test smb.version keyword
```

Comment:
Could you please add an indication of where the pcap used here comes from? :)

Reply:
In this case where I obtained it from the original submitters branch, it appears to be a capture from a local network they had access to. I am not sure how to denote that. How would you like that documented?

https://github.com/zer1t0/suricata-verify/tree/test/smb_version

Reply:
Fair enough for me

Reply:
Sorry for the late reply. A note with something along those lines - even if just - 'sample traffic from a local network' works, IMHO. :)